src/main/webapp/queue/ta.js

```diff
@@ -82,12 +82,12 @@ function getQueue() {
         queueListElement.appendChild(createListElement(studentEmail));
     });
 
+    const beinghelpedElem = document.getElementById('beingHelped');
```

Comment:
I think I saw this change in both Bianca and Nick's CLs, make sure to coordinate with them